woocommerce/woocommerce
# Remove homepage template prompt logic

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #40426  .

This PR disables homepage template prompt implemented in https://github.com/woocommerce/woocommerce/pull/40363 temporarily for MVP.



### How to test the changes in this Pull Request:

This feature is part of the larger customize your store development and behind a feature flag, hence QA review can be deferred until the call for testing before release.

Make sure you are testing this with a Jetpack connected instance of WooCommerce, as the AI text completion endpoint can only be called with a Jetpack connected account.


1. Create a new WooCommerce installation with this version.
2. Make sure to enable `customize-store` feature flag
3. Go to `WooCommerce -> Home` and click `Customize Store` task
4. Click `Design with AI` button
5. Fill in the store description and click the continue button.
6. Proceed all steps until the end
7. You should be redirected to assember hub.
8. Click on `Change your homepage`
9. Confirm none of the six templates is selected or shown on the preview.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Remove homepage prompt logic from CYS

#### Comment 

</details>
